src/PasswordGenerator.h

```diff
@@ -0,0 +1,17 @@
+// Copyright 2005-2017 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_PASSWORDGENERATOR_H_
+#define MUMBLE_PASSWORDGENERATOR_H_
+
+#include <QString>
+
+class PasswordGenerator {
+	public:
+		static QString alphabet();
```

Comment:
Too bad there's no revision diff where *I* could spot this as well, without looking through everything again :)